src/pages/Home.jsx

```diff
@@ -0,0 +1,69 @@
+import React from "react";
+import { useState, useEffect } from 'react'
+import axios from 'axios'
+import '../Styles/Home.css'
+
+
+const Home = () => {
+    
+    
+    const [homeArticles, setHomeArticles] = useState([])
+
+
+    useEffect(() => {
+        axios.get('https://nc-news-fz7g.onrender.com/api/articles')
+        .then(({data}) => {
+        console.log(data, '<<< consoling the useEffect')
+        setHomeArticles(data.articles)
+      
+    })
+    .catch((error)=>{
+    console.log('Error fetching articles:', error)
+  })
+}, [])
+
+console.log(homeArticles, '<<<<home articles')
+
+const calculateTimeAgo = (timePosted) => {
+    const currentTime = new Date()
+    const postedTime = new Date(timePosted)
+    const differenceMs = currentTime - postedTime
+    const differenceDays = Math.floor(differenceMs / (1000 * 60 * 60 * 24))
+    if (differenceDays > 365) {
+        return (Math.floor(differenceDays/365)) + ` years ago`
+    }
+    return differenceDays + ` days ago`
```

Comment:
If you wanted a lightweight library to do this kind of logic for you, you could use `Day.js`, here's their site https://day.js.org/